packages/react-storage/src/components/StorageBrowser/views/Controls/Paginate.tsx

```diff
@@ -81,15 +75,13 @@ const getButtonVariantProps = (
 // `getButtonVariantProps` instead of a React component
 export interface PaginateControlProps {
   // eslint-disable-next-line react/no-unused-prop-types
-  currentPage?: number;
+  currentPage: number;
```

Comment:
No required props on components, else consumers will get TS errors when composing. For example, the below code would have TS errors for missing properties:
```
<StorageBrowser.LocationsView.Paginate />
```

Reply:
Fixed